src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -140,15 +147,27 @@ const NavBar: React.FC<NavBarPropsType> = ({
   }, [radioFirstPlayStateLoaded]);
 
   const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
-  const toggleEventSchedule = useCallback(() => {
+  function toggleEventSchedule() {
+    console.log("toggling");
     setEventScheduleVisible(!isEventScheduleVisible);
-  }, [isEventScheduleVisible]);
+  }
   const hideEventSchedule = useCallback((e) => {
     if (e.target.closest(`.${navBarScheduleClassName}`)) return;
-
     setEventScheduleVisible(false);
   }, []);
```

Comment:
Why was the newline removed from here? We keep a blank line after our 'guard statements' like this

Reply:
readded, sorry ;-)

Reply:
All good. Thanks :)